spec.bs

```diff
@@ -529,6 +691,36 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
         1. If |workletGlobalScope| has an associated boolean [=addModule success=], set |workletGlobalScope|'s [=addModule success=] to true.
         2. [=Resolve=] |promise|.
 
+  Just before the final step, currently "Return <var ignore>promise</var>.", add the following step:
+
+    7. If |this| is a {{SharedStorageWorklet}}, [=upon fulfillment=] of |promise| or
```

Comment:
Nit: It seems that currently "`Return <var ignore>promise</var>.`" is numbered 6, so if you want this step to precede it, your new step would need to be numbered 6.

Reply:
We add an additional step above (the new step 4), so I think it would be step 7 after that.

Reply:
You're right, I missed that.  Thanks!